parallel-twitter-trending-hashtags/app.py

```diff
@@ -0,0 +1,38 @@
+from flask import Flask,jsonify,request
+from flask import render_template
+import ast
+
+
+app = Flask(__name__)
+
+hashtags = []
+counts = []
+
+
+@app.route("/")
+def get_chart():
+    global hashtags,counts
+    hashtags = []
+    counts = []
+    return render_template('index.html', counts_data=counts, hashtags_data=hashtags)
+
+
+@app.route('/refreshDashboard')
+def refresh_graph_data():
+    global hashtags, counts
+    return jsonify(r_hashtags=hashtags, r_counts=counts)
+
+
+@app.route('/updateDashboard', methods=['POST'])
+def update_data_post():
+    global hashtags, counts
+    if not request.form or 'data' not in request.form:
+        return "error: no data",400
+    hashtags = ast.literal_eval(request.form['label'])
```

Comment:
This is parsing json, it should be using the `json` library, not the ast library. This applies to all uses of the ast library in this code.

Reply:
This is from Hannee's code. I didn't touch anything in that file. I'm not a very good python programmer so I'm not sure if I know how to use the `json` library but can take a look.

Reply:
I agree with @aturley